.github/workflows/test-and-build.yml

```diff
@@ -63,9 +62,50 @@ jobs:
           QT_QPA_PLATFORM: offscreen
 
 
-  build:
+  # TODO: Consider extracting everything below this line to a separate workflow
+  # which is triggered by pushes to the default branch and GitHub releases.
+  build-and-release-image:
     runs-on: "ubuntu-latest"
     needs: ["test"]
+    if: "github.ref_type == 'tag' || github.ref_type == 'branch'"
+    env:
+      IMAGE_NAME: "nsidc/qgreenland"
+      IMAGE_TAG: "${{ github.ref_type == 'branch' && 'latest' || github.ref_name }}"
```

Comment:
Conditionals aren't well supported. This is a ternary expression. Re-written as Python: `'latest' if github.ref_type == 'branch' else github.ref_name`

Reply:
e38c9d87